EIPS/eip-3664.md

```diff
@@ -0,0 +1,383 @@
+---
+eip: 3664
+title: Next generation metaverse NFT
+description: This is a standard for next generation metaverse NFTs, which extends ERC-1155.
+author: TylerZ (@tylerztl), TY (@nimocat)
+discussions-to: https://ethereum-magicians.org/t/eip-3664-next-generation-game-nft-standard/6738
+status: Draft
+type: Standards Track
+category: ERC
+created: 2021-07-17
+requires: 20, 1155
+---
+
+## Abstract
+In this proposal, we created the next gen component-NFT standard, which makes up for the problem that [ERC-721](./eip-721.md) and [ERC-1155](./eip-1155.md) can only support unchangeable attributes of NFTs. component-NFT can make the attributes in a NFT have five characteristics: transferable, upgradeable, changeable, combinable and evolvable. By using this standard NFTs in any fields, users can create any valuable dapps based on component-NFT standard.
+
+## Motivation
+NFT standards such as ERC721 and ERC1155 are now widely recognized and utilized, but these two protocols lack descriptive power, particularly regarding metadata. As GameFi and Layer 2 solutions emerge and gain popularity, a large of number of avatars, props, etc. are issued as NFT. However, their growth are limited by the nature of existing NFT protocols. It is well known that characters and props typically consist of complex attributes which can change over time. These attributes enrich characters and props, hence changing their value. However, ERC721 and ERC1155 express these attributes via tokenURI, which are typically fixed URL addresses or descriptive texts. Obviously these do not suffice for the mutable attributes required by characters and props.
+
+In our current proposal, we suggest a new type of extendable NFT metadata protocol, which is compatible with existing ERC721 and ERC1155, and can build on their basis to provide better descriptive power for attributes metadata.
+
+Currently we provide the following options for attribute extendability:
+1.	General attributes: for describing immutable attributes such as avatar birth dates.
+2.	Variable attributes: for describing attributes whose value will change, such as an avatar’s combat power.
+3.	Transferable attributes: for describing attributes that can be transferred to other NFTs.
+4.	Upgradable attributes: for describing NFT levels and can trigger upgrades.
+5.	Evolvable attributes: for describing the fact that NFT can evolve, and evolutions can fail, and if the case of failure, the NFT can no longer be used until properly repaired.
+6.	Text attributes: for realizing functionality similar to Loot  since the above attributes are numerical in nature. 
```

Comment:
This is less motivation, and more specification I think?

Reply:
Rationale maybe?  Describing *why* each field is useful is a rationale thing.  Listing standard fields is a specification thing.  Explaining why the EIP in general is useful is a motivation thing.